packages/amplication-server/src/core/billing/billing.service.ts

```diff
@@ -264,6 +264,7 @@ export class BillingService {
       const stiggClient = await this.getStiggClient();
       await stiggClient.provisionCustomer({
         customerId: workspaceId,
+        shouldSyncFree: false,
```

Comment:
Missing unit test

Reply:
It's a request to external service. It can't be tested by unit test.

Reply:
it can, the amplication business logic is to not sync free account, so we can spy and check that we pass the correct arguments.

Reply:
Currently we don't have any tests for BillingService. Mocking it's dependencies and writing basic tests will take time. I've opened a separate issue form that: https://github.com/amplication/amplication/issues/5539
Let's move on with this minor change and test in manually.